### DISCUSSION TITLE
Hosting on decentralized storage

### DISCUSSION THREAD

**Discussion by nemanjakrstonic** [11ty/eleventy#3058] [Q&A]
Hi, I’m looking to host my website on decentralized storage. So far, I found this service [peerspace.network](https://peerspace.network/?utm_source=github&utm_medium=Message&utm_campaign=decentralized_storage&utm_id=User+discovery+) that offers hosting and it's similar to Netlify but on Arweave. 

Has anybody tried to migrate to decentralized storage? What was your experience?

**Comment by Snapstromegon** [comment]
I haven't tried this yet, but I personally would not feel comfortable to host anything on the arweave.net domain (or a domain CNAMEd to it) as long as that domain is not part of the publicsuffix list https://publicsuffix.org/ like github.io. This is, because as long as the domain is not on that list, modern browsers will not separate pages as they should be.
